R/centrality.R

```diff
@@ -1177,6 +1178,13 @@ bonpow.sparse <- function(graph, nodes = V(graph), loops = FALSE,
 #'   loops.  `loops` is `FALSE` by default.
 #' @param exponent exponent (decay rate) for the Bonacich power centrality
 #'   score; can be negative
+#' @param weights A numerical vector or `NULL`. This argument can be used
+#'   to give edge weights for calculating the weighted eigenvector centrality of
+#'   vertices. If this is `NULL` and the graph has a `weight` edge
+#'   attribute then that is used. If `weights` is a numerical vector then it is
+#'   used, even if the graph has a `weight` edge attribute. If this is
+#'   `NA`, then no edge weights are used (even if the graph has a
+#'   `weight` edge attribute).
```

Comment:
This describes the behaviour we want, but the function does not currently follow it.

Reply:
This text still refers to the eigenvector centrality, that should be fixed.